src/main/java/org/littil/api/coordinates/service/SearchService.java

```diff
@@ -0,0 +1,19 @@
+package org.littil.api.coordinates.service;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.QueryParam;
+import java.util.Set;
+
+@Path("/search")
```

Comment:
I don't think we have a need for a public endpoint to fetch coordinates. Only in the background.

Reply:
I don't think this creates a public endpoint using @RegisterRestClient. However, I've removed /search here and appended it to the base url in application.yaml